LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2019 Exercism
+Copyright (c) 2022 Exercism
```

Comment:
Very interesting read. As I'm reading it, changing the existing year in the license to the current year is discouraged?

Reply:
I think especially here, where things are version controlled, we can a bit more easily show which portions are copyright events.  So it may be appropriate to add dates, but I would not change the date, outright, only append a range or a new year, depending on change activity.